### ISSUE TITLE
TIMESTAMP() returns error  , timestamp() does not

### ISSUE THREAD

**Issue by tomzeppenfeldt** [neo4j/neo4j#12097]
- Neo4j version: 3.5.0
- Operating system: (Mac OS)

- In Neo4j Browser

```
RETURN TIMESTAMP()    
RETURN timestamp()
```
    
- Expected behavior

Both should work

- Actual behavior

```
RETURN  TIMESTAMP()   
````
returns error

```
Neo.ClientError.Statement.SyntaxError: Unknown function 'TIMESTAMP' (line 1, column 8 (offset: 7))
"return TIMESTAMP()"
```

**Comment by Dcanzano** [comment]
@ophileon  thanks for reporting and i have also reproduced using bin/cypher-shell such that 'return timestamp();` succeeds but `return TIMESTAMP();`  errors.   ill update the cypher team

**Comment by pontusmelke** [comment]
@ophileon Thanks for reporting this. The issue has been fixed and will be part of the next 3.5 release.

Thanks again!
